packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -917,4 +917,74 @@
       <protected>false</protected>
       <shortDescription>Show less</shortDescription>
     </labels>
+    <labels>
+      <fullName>quantic_ExplainWhy</fullName>
+      <value>Explain why</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Explain why</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_SelectTheReason</fullName>
+      <value>Select the reason</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Select the reason</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_WhatWasTheReason</fullName>
+      <value>What was the reason?</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>What was the reason?</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_FillOutField</fullName>
+      <value>Please fill out this field</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Please fill out this field</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_SelectOneOfOptions</fullName>
+      <value>Please select one of these options</value>
```

Comment:
Since all the options are highlighted in red, you could just say "Please select an option"